### DISCUSSION TITLE
images are pixelated

### DISCUSSION THREAD

**Discussion by chaiyiii** [apache/netbeans#5972] [Q&A]
Hi, I just wanna ask why my netbeans images are pixelated, also I am new to netbeans, I have been searching all over and can't seem to find answers. I hope anyone can help. 
![image](https://github.com/apache/netbeans/assets/134152231/45cd8e78-1c49-4aeb-b08b-6fba295a85b4)
![image](https://github.com/apache/netbeans/assets/134152231/2e762869-eec9-4c71-a8c3-2bebc607bbcc)

**Comment by chaiyiii** [comment]
solved.

**Reply by asbachb** [reply]
Feel free to post the solution to your problem :)
